lib/ZcapClient.js

```diff
@@ -138,14 +145,15 @@ export class ZcapClient {
 
     const signedDelegatedCapability = await jsigs.sign(
       delegatedCapability, {
-        suite: new Ed25519Signature2018({
+        suite: new Ed25519Signature2020({
           signer: this.delegationSigner,
           verificationMethod: this.delegationSigner.id
```

Comment:
This doesn't exist anymore: https://github.com/digitalbazaar/ed25519-signature-2020/blob/main/lib/Ed25519Signature2020.js#L45

And it looks like we can just drop it given that `id` is set on the signer: https://github.com/digitalbazaar/jsonld-signatures/blob/8a4d36ed9d4167dde5ecfe30051361115806284c/lib/suites/LinkedDataSignature.js#L344-L345

```suggestion
```